### PR TITLE
Create a new plugin for headline block

### PR DESCRIPTION
**Pull request by fuerzastudio**

### CODE REVIEW

**Comment by cjyabraham** [comment]
Can you add the css to the theme css file rather than delivering it in a separate file associated with this plugin?  I'd prefer to avoid having another css file.

**Comment by fuerzastudio** [comment]
> Can you add the css to the theme css file rather than delivering it in a separate file associated with this plugin? I'd prefer to avoid having another css file.

OK. Done!

**Comment by cjyabraham** [comment]
@andycochran yes, we should stay consistent with how we've been doing things, so not use variables.  Are there other changes you'd recommend?

File: web/wp-content/themes/lfevents/src/assets/scss/modules/_alert.scss [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+    opacity: 1;
+    padding: 10px;
+    text-align: center;
+    width: 100%;
+
+    a {
+      color: var(--text-color);
+      font-weight: bold;
+      text-decoration: underline;
+    }
```
Comment (andycochran) [inline web/wp-content/themes/lfevents/src/assets/scss/modules/_alert.scss]: Can this just be… 
``` 
a {
  color: inherit;
  text-decoration: underline;
}
```

There's no reason to impose the bold.

File: web/wp-content/themes/lfevents/src/assets/scss/modules/_alert.scss [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+  max-width: none;
+  width: 100%;
+
+  p {
+    -webkit-font-smoothing: auto;
+    font-size: 18px;
+    line-height: 1.2;
+    margin: 0 auto;
+    max-width: 1279px;
+    opacity: 1;
```
Comment (andycochran) [inline web/wp-content/themes/lfevents/src/assets/scss/modules/_alert.scss]: Why do we need opacity here?

File: web/wp-content/themes/lfevents/src/assets/scss/modules/_alert.scss [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+  color: var(--text-color);
+  margin: 0 auto;
+  height: auto;
+  margin-bottom: 1.5rem;
+  max-width: none;
+  width: 100%;
+
+  p {
+    -webkit-font-smoothing: auto;
+    font-size: 18px;
```
Comment (andycochran) [inline web/wp-content/themes/lfevents/src/assets/scss/modules/_alert.scss]: We don't set font sizes in pixels. We've set them in relative units (rem) throughout the site.

File: web/wp-content/themes/lfevents/src/assets/scss/modules/_alert.scss [hunk 4]
```diff
@@ -0,0 +1,29 @@
...
+  --text-color: #212326;
+  background-color: var(--bg-color);
+  color: var(--text-color);
+  margin: 0 auto;
+  height: auto;
+  margin-bottom: 1.5rem;
+  max-width: none;
+  width: 100%;
+
+  p {
```
Comment (andycochran) [inline web/wp-content/themes/lfevents/src/assets/scss/modules/_alert.scss]: Do we need explicit paragraph styles? What if the user adds an `h3`/etc instead?

File: web/wp-content/themes/lfevents/src/assets/scss/modules/_content.scss [hunk 5]
```diff
@@ -36,6 +36,8 @@
   // Full width template
   .main-content-full-width {
     @include xy-cell(12);
+    // width: 100%;
+    // margin: 0 auto;
   }
```
Comment (andycochran) [inline web/wp-content/themes/lfevents/src/assets/scss/modules/_content.scss]: Can we nix these comments?

**Comment by rodmeurer** [comment]
@andycochran @cjyabraham  up to date. Let us know if you have any issue on this.

**Pull request merged**